### PR TITLE
fix 4-projects uat feedback issues

### PR DESCRIPTION
**Pull request by daniel-cit**
Fixes https://github.com/terraform-google-modules/terraform-example-foundation/issues/444

### CODE REVIEW

File: 4-projects/development.auto.example.tfvars [hunk 1]
```diff
@@ -15,3 +15,5 @@
  */
 
 perimeter_name = "sp_d_shared_restricted_default_perimeter_????"
+
+//enable_hub_and_spoke = true
```
Comment (rjerrems) [inline 4-projects/development.auto.example.tfvars:19]: Should this not go in the common tfvars file so the user does not have to set it three times?

**Review by rjerrems — APPROVED** [review]
LGTM - one minor nit around tfvars.

/cc @bharathkkb

File: 4-projects/development.auto.example.tfvars (hunk 1 above)
Comment (daniel-cit) [reply 4-projects/development.auto.example.tfvars:19]: @rjerrems If we add it in the commons tfvars files we get a warning that we are setting a variable that does not exist when running the shared envs for bu1 and bu2 .

Comment (rjerrems) [reply 4-projects/development.auto.example.tfvars:19]: aha got it, thanks!

**Pull request merged**